e2etests/vuln_test.go

```diff
@@ -142,23 +142,16 @@ func TestStackroxVulnImages(t *testing.T) {
 			image: "quay.io/rhacs-eng/qa:rails-cve-2016-2098",
 			expectedFeatures: []feature{
 				{"rails", "4.2.5.1", []expectedVuln{
+					{name: "CVE-2016-2098"},
+					{name: "CVE-2016-6316"},
+					{name: "CVE-2016-6317"},
```

Comment:
🤔  - these are still marked as deferred in NVD, were these moved because the 2.0 feed includes them again?